File: WorldPlants/Models/GetUserSiteSettingsDto.cs
// Ignore Spelling: Dto

namespace WorldPlants.Models
{
    public class GetUserSiteSettingsDto
    {
        public string Name { get; set; }
        public int WarmPeriodMinTemperature { get; set; }
        public int WarmPeriodMaxTemperature { get; set; }
        public int ColdPeriodMinTemperature { get; set; }
        public int ColdPeriodMaxTemperature { get; set; }
        public bool HasRoof { get; set; }
        public bool CanChangeHasRoof { get; set; }
        public int Location { get; set; }
        public int SunExposureId { get; set; }
    }
}

File: WorldPlants/ClientApp/src/Interfaces/GetUserSiteSettingsDto.ts
export interface GetUserSiteSettingsDto {
  name: string;
  warmPeriodMinTemperature: number;
  warmPeriodMaxTemperature: number;
  coldPeriodMinTemperature: number;
  coldPeriodMaxTemperature: number;
  hasRoof: boolean;
  canChangeHasRoof: boolean;
  location: number;
  sunExposureId: number;
}
